overrides/contracts/blackout/Blackout_DugInDeep.json

```diff
@@ -462,7 +462,7 @@
       "overrideDialogueBucketId": "",
       "dialogueContent": [
         {
-          "words": "{COMPANY.CompanyName}! We just watched you waste that invading lance. What a sight for sore eyes. Welcome to the frontlines!",
+          "words": "{COMPANY.CompanyName}! We just watched you waste those compromised turrets. What a sight for sore eyes. Welcome to the frontlines!",
```

Comment:
Due to the nature of `Blackout`, the first event point has 3 events in it.

* Nothing happens - your lance detect _something_ happened but nothing really happens
* The turrets activate for a reason and attack you
* You get ambushed by a lance of mechs who drop on you with drop pods, and the turrets activate to defend you

Changing the dialogue here to be turrets isn't correct. At the same time, `that invading lance` is also incorrect for the the same reason. Maybe it can be changed to something that covers the three scenarios in a good but generic way.

Reply:
Updated with more generic text.